src/services/device.js

```diff
@@ -46,41 +47,53 @@ function getHistory(readings) {
 }
 
 export function getReadings(deviceUID) {
-  var eightDaysAgo = new Date(new Date().setDate(new Date().getDate() - 8));
-
-  return fetch('https://airnote-api.onrender.com/?' + new URLSearchParams({
-    device_uid: deviceUID,
-    from: eightDaysAgo.toISOString(),
-    to: new Date().toISOString()
-  }))
-    .then(response => response.json())
-    .then(data => {
+  return fetch(
+    "http://localhost:3001/?" +
```

Comment:
Will this hardcoded URL work in prod or does it need to be more dynamic?

Reply:
Ah yeah I totally forgot about this. It definitely needs to be dynamic. I’ll take a stab at this today.

Reply:
Ok here’s my fix for that ad03d27390ce2fe66624f1956c85ea6aca5c3f3f.

The Prettier auto formatting makes this messy but the key part is this bit in `rollup.config.js`. This substitution was already happening for the Notehub API which made adding this one pretty easy.

```
      "process.env.SERVER_URL": JSON.stringify(
        production
          ? "https://airnote-api.onrender.com"
          : "http://localhost:3001"
      ),
```